src/main/java/org/folio/rest/impl/InventoryHelper.java

```diff
@@ -450,14 +397,13 @@ private Error buildErrorWithParameter(String value, ErrorCodes errorCode) {
       .withParameters(parameters);
   }
 
-  private String buildProductIdQuery(ProductId productId, Map<String, String> productTypes) {
-    return String.format("(identifiers adj \"\\\"identifierTypeId\\\": \\\"%s\\\"\" " +
-        "and identifiers adj \"\\\"value\\\": \\\"%s\\\"\")",
-      productTypes.get(productId.getProductIdType().toString()),
-      productId.getProductId());
+  private String buildProductIdQuery(ProductId productId) {
+    return String.format(
+        "(identifiers adj \"\"identifierTypeId\": \"%s\"\" \" + and identifiers adj \"\"value\": \"%s\"\")",
```

Comment:
This will produce invalid CQL query. Lets say I would like to search for some instances which has
```json
...
{
  "value": "0552142352",
  "identifierTypeId": "8261054f-be78-422d-bd51-4ed9f33c3422"
}
...
```

The string above will be transformed to and the search would fail
```
(identifiers adj ""identifierTypeId": "8261054f-be78-422d-bd51-4ed9f33c3422"" " + and identifiers adj ""value": "0552142352"")
```
![image](https://user-images.githubusercontent.com/43410167/59117994-cca47200-8957-11e9-8068-da1616409751.png)

The valid query is  
```
query=(identifiers adj "\"identifierTypeId\": \"8261054f-be78-422d-bd51-4ed9f33c3422\"" and identifiers adj "\"value\": \"0552142352\"")
```
![image](https://user-images.githubusercontent.com/43410167/59117912-8fd87b00-8957-11e9-98c3-91bb1db8a080.png)

Please leave the version from master

Reply:
Ok. Left as is.